modules/ROOT/pages/dataweave-formats-dw.adoc

```diff
@@ -1,103 +1,64 @@
-= DataWeave Format (dw)
-// :page-aliases: 4.3@mule-runtime::dataweave-formats-dw.adoc
 
-MIME Type: `application/dw`
+= Dw Format
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: format, dw, */dw, application/dw
+
+MIME type: `application/dw`
 
 ID: `dw`
 
-The DataWeave (dw) format is the canonical format for all transformations.
-This format can help you understand how input data is interpreted before it is
-transformed to a new format.
-
-[NOTE]
-====
-This format is intended to help you debug the results of DataWeave transformations. It is significantly slower than other formats. It is not recommended to be used in production applications because it can impact the performance.
-====
-
-[[examples]]
-== Example: Express XML in the DataWeave (dw) Format
-
-The following example shows how to transform XML input to the DataWeave (dw) format.
-
-=== Input
-
-The example uses the following XML snippet as input.
-
-[source,xml,linenums]
-----
-<employees>
-  <employee>
-    <firstname>Mariano</firstname>
-    <lastname>DeAchaval</lastname>
-  </employee>
-  <employee>
-    <firstname>Leandro</firstname>
-    <lastname>Shokida</lastname>
-  </employee>
-</employees>
-----
-
-==== Output
-
-.Output: in DataWeave Format
-[source,dataweave,linenums]
-----
-{
-  employees: {
-    employee: {
-      firstname: "Mariano",
-      lastname: "DeAchaval"
-    },
-    employee: {
-      firstname: "Leandro",
-      lastname: "Shokida"
-    }
-  }
-} as Object {encoding: "UTF-8", mimeType: "text/xml"}
-----
-
-
-// CONFIG PROPS ///////////////////////////////////////////////////////
+
```

Comment:
TODO: looks like we might be missing a docs.asciidoc for the DW format.
@mlischetti FYI. I'll add to Jira.

Reply:
/DW-757